lib/galaxy/util/__init__.py

```diff
@@ -1481,9 +1481,9 @@ def send_mail(frm, to, subject, body, config, html=None):
 
     smtp_ssl = asbool(getattr(config, 'smtp_ssl', False))
     if smtp_ssl:
-        s = smtplib.SMTP_SSL()
+        s = smtplib.SMTP_SSL(config.smtp_server)
     else:
-        s = smtplib.SMTP()
+        s = smtplib.SMTP(config.smtp_server)
     s.connect(config.smtp_server)
```

Comment:
The `connect()` is called by `SMTP()`/`SMTP_SSL()` if the `host` parameter is passed, so this line shouldn't be needed any more.

```suggestion
```